jaeger/model/keyvalue.go

```diff
@@ -210,7 +210,7 @@ func (kvs KeyValues) Hash(w io.Writer) error {
 }
 
 // Hash implements Hash from Hashable.
-func (kv KeyValue) Hash(w io.Writer) error {
+func (kv *KeyValue) Hash(w io.Writer) error {
```

Comment:
This is the only thing we should look at doing. I'm not sure if pass-by-reference would be valid here in this use case, but typically, pass-by-reference is what we want.

My suggestion - strip everything out of this PR of the useless churn (changing uppercase to lowercase and punctuation in log messages) and keep useful changes like this. Then we can review further.

Reply:
Thanks @jmazzitelli, I will update it later :)

Reply:
I would also upvote the changes to remove the unnecessary fmt.Sprintf instances.

Reply:
Could we move forward? PR updated. @jmazzitelli @jshaughn